### ISSUE TITLE
training loss problem

### ISSUE THREAD

**Issue by godchengzhihang** [JiahuiYu/generative_inpainting#314]
Hello, when I trained myself on the celeba dataset, my generator loss first dropped normally, but it went up until 50,000 steps,And when I tested the image in the example, the result was very bad.Can you help me?

**Comment by godchengzhihang** [comment]
@JiahuiYu How many times do you need to train on the celeba dataset? Why is the value of the loss function that is later in my training getting bigger and bigger?I hope to get your guidance and help!

**Comment by JiahuiYu** [comment]
@godchengzhihang It should be normal for training GANs - you need more training I guess.

**Comment by LeonCurry** [comment]
I have a question about training loss.  What is the loss which is displayed in the Pycharm interface during the training process? generator loss or critic loss or something else? @JiahuiYu
